src/dialog/dialog.scss

```diff
@@ -0,0 +1,207 @@
+    top: 33%;
+    transform: translate(-50%, -33%);
+    max-width: 50%;
+    min-width: 380px;
+    max-height: 100vh;
+
+    @media screen and (max-width: 400px) {
+      max-width: 95%;
+      width: 95%;
+      min-width: 95%;
+    }
+  }
+}
+
+@mixin iui-dialog-full-page {
+  @media (prefers-reduced-motion: no-preference) {
+    transition: visibility $iui-speed-instant linear $iui-speed-slow, opacity $iui-speed ease-out $iui-speed-fast;
+  }
+
+  .iui-dialog {
+    border-radius: 0;
+    height: 100vh;
+    width: 100vw;
+    top: 0;
+    left: 0;
+    max-width: initial;
+    min-width: initial;
+    display: flex;
+    flex-direction: column;
+    will-change: transform;
+    @media (prefers-reduced-motion: no-preference) {
+      transform: translateY(100%);
+      transition: visibility $iui-speed-instant linear $iui-speed, opacity $iui-speed-instant linear $iui-speed,
+        transform $iui-speed-fast ease-in;
+    }
+  }
+
+  &.iui-dialog-visible .iui-dialog {
+    transform: translateY(0);
+    @media (prefers-reduced-motion: no-preference) {
+      transition: transform $iui-speed ease-out;
+    }
+  }
+}
+
+@mixin iui-dialog-draggable {
+  background-color: transparent;
+  pointer-events: none;
+  z-index: 998;
+
+  .iui-dialog {
+    pointer-events: initial;
+    max-width: 100vw;
+    max-height: 100vh;
+    min-width: 380px;
+    min-height: 144px;
+    resize: both;
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+    @include themed {
+      border: 1px solid t(iui-color-background-border);
+    }
+  }
+}
+
+@mixin iui-dialog-animation {
+  // Full Page Dialog Animations for React
+  &-enter .iui-dialog-full-page .iui-dialog {
+    transform: translateY(100%);
+    opacity: 0;
+  }
+
+  &-enter-active .iui-dialog-full-page .iui-dialog {
+    transform: translateY(0);
+    opacity: 1;
+  }
+
+  // Small Dialog Animations for React
+  &-enter .iui-dialog-backdrop {
+    visibility: hidden;
+    opacity: 0;
+  }
+
+  &-enter-active .iui-dialog-backdrop {
+    visibility: visible;
+    opacity: 1;
+  }
+}
+
+@mixin iui-dialog-title {
+  font-size: inherit;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  line-height: 1.5;
+}
+
+@mixin iui-dialog-title-bar {
+  display: flex;
+  align-items: center;
+  margin-bottom: $iui-baseline;
+  justify-content: space-between;
+  box-sizing: border-box;
+  font-size: $iui-font-size-subheading;
+
+  @at-root .iui-dialog-draggable & {
+    user-select: none;
+    font-size: $iui-font-size-leading;
+    padding: round($iui-baseline * 0.5) $iui-m;
+    cursor: grab;
+    @include themed {
+      background-color: t(iui-color-background-3);
+      border-bottom: 1px solid t(iui-color-background-border);
+    }
+
+    &:active {
+      cursor: grabbing;
+    }
+  }
+}
+
+@mixin iui-dialog-content {
+  flex-grow: 2;
+  margin: 0 -#{$iui-m};
+  padding: 0 $iui-m;
+  overflow-y: auto;
+  overflow-y: overlay;
+
+  @at-root .iui-dialog-draggable & {
```

Comment:
Why it is different for draggable?

Reply:
When draggable we change a few things:
- Cursor change to `grab`.
- The title bar area gets a bar area to grab from.
- `user-select: none` so they don't accidentally highlight text when trying to drag.

Reply:
No, I mean these two rules. I understand that draggable header is styled differently 😀

Reply:
Was able to remove 1 line of duplicate CSS.  We discussed the rest in a chat, should be good to go now.